pkg/operator/ceph/controller/spec.go

```diff
@@ -615,6 +634,51 @@ func StoredLogAndCrashVolumeMount(varLogCephDir, varLibCephCrashDir string) []v1
 	}
 }
 
+// GenerateMDSLivenessProbeExecDaemon generates a liveness probe that makes sure mds daemon is present in fs map,
+// that it can be called, and that it returns 0
+func GenerateMDSLivenessProbeExecDaemon(daemonName string) *v1.Probe {
+	daemonName = strings.TrimLeft(daemonName, "rook-ceph-mds-")
+	var mdsLivenessProbeCmd = []string{
+		"timeout",
+		fmt.Sprintf("%d", mdsCmdTimeout),
+		"sh",
+		"-c",
+		fmt.Sprintf(`outp="$(ceph fs dump --mon-host=$(ROOK_CEPH_MON_HOST)
+		--mon-initial-members=$(ROOK_CEPH_MON_INITIAL_MEMBERS) --format json 2>&1)"
```

Comment:
Can we dump only stats for a specific filesystem -- the one for this MDS -- to keep the json return list small? Otherwise, jq needs to iterate over all fs'es in the map.

Reply:
Do we have a cms for that, that link filesystem with mds

Reply:
The list still need to be fetched, to get the standbys mds